packages/files-ui/cypress/support/page-objects/basePage.ts

```diff
@@ -2,6 +2,7 @@
 
 export const basePage = {
   appHeaderLabel: () => cy.get("[data-cy=files-app-header]", { timeout: 20000 }),
+  searchInput: () => cy.get("[data-testid=input_searchbar]"),
```

Comment:
Following previous suggestion
```suggestion
  searchInput: () => cy.get("[data-testid=input-search-bar]"),
```